### DISCUSSION TITLE
Feature Request: Add Exploit mitigation techniques and Sandbox App for Greater Security

### DISCUSSION THREAD

**Discussion by ghost** [File-New-Project/EarTrumpet#1405] [Completed ideas]
If possible you could enable exploit mitigations such as CFG, ACG and CET.
This way it will get rid of classes of exploit vulnerabilities.

Microsoft is creating a new sandbox environment that is easy to adapt and implement which will strengthen the security even further.

For more information:
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/customize-exploit-protection?view=o365-worldwide
https://github.com/microsoft/win32-app-isolation

**Comment by riverar** [comment]
Hi @xAzazellx, EarTrumpet is written in C# and relies on .NET. It is up to them to compile with CFG, ACG, and CET mitigations turned on. Here's their full roadmap on these https://github.com/dotnet/designs/blob/main/accepted/2021/runtime-security-mitigations.md#windows-control-flow-guard-cfg

Regarding Win32 App Isolation, thanks. We're aware and have been in contact with that team for a year or so now. I'm also actively participating in that repository (e.g., https://github.com/microsoft/win32-app-isolation/issues/31).

Going to close this for now as there's no work to be done. Feel free to keep the discussion going!
